doc/content/reference/email-templates/available.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Available Templates"
+description: ""
+weight: 3
+---
+
+The following email templates are currently used by The Things Stack:
+
+Full Name | Identifier | Scope | Additional fields
+----------|------------|-------|------------------
+API Key changed | api_key_changed | Sent when the rights of an API Key have been changed. | `Identifiers` and `Rights`
+API Key created | api_key_created | Send when an API Key has been created. | `Identifiers` and `Rights`
+Collaborator changed | collaborator_changed | Sent when the rights of a collaborator have been changed. | `Collaborator`
+Password changed | password_changed | Sent when the the password of an user has been changed.
+Temporary password | temporary_password | Sent when an temporary password has been requested for an user. | `TemporaryPassword`
+Account validation | validate | Sent when a user is created in order to validate their email address. | `ID` and `Token`
```

Comment:
This is not just for accounts, also for other (app,cli,gtw,org) contact info of EMAIL type